app/controllers/concerns/hyku_addons/work_factory_behavior.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
```

Comment:
I know this isn't where this file goes but its not loading from lib so I put it here till I can figure out how to get it working in the engine from lib